src/jvmTest/java/org/snakeyaml/engine/issues/issue149/GlobalDirectivesTest.java

```diff
@@ -1,71 +0,0 @@
-/*
- * Copyright (c) 2018, SnakeYAML
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the License
- * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied. See the License for the specific language governing permissions and limitations under
- * the License.
- */
-package org.snakeyaml.engine.issues.issue149;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-import java.io.InputStream;
-import java.util.concurrent.atomic.AtomicInteger;
-
-import org.intellij.lang.annotations.Language;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
-import it.krzeminski.snakeyaml.engine.kmp.api.LoadSettings;
-import it.krzeminski.snakeyaml.engine.kmp.api.lowlevel.Parse;
-import it.krzeminski.snakeyaml.engine.kmp.events.Event;
-import it.krzeminski.snakeyaml.engine.kmp.exceptions.ParserException;
-import org.snakeyaml.engine.v2.utils.TestUtils;
-
-@org.junit.jupiter.api.Tag("fast")
-public class GlobalDirectivesTest {
-
-  Iterable<Event> yamlToEvents(@Language("file-reference") final String resourceName) {
```

Comment:
note: `org.intellij.lang.annotations.Language` is now available for KMP. We probably could try to keep those annotations by adding a new dependency to tests

```kotlin
testImplementation("org.jetbrains:annotations:26.0.1")
```

Reply:
Tried it quickly: we'd have to adjust how we refer for the resource files. The current usage leads to inability to resolve the path:
<img width="718" alt="Screenshot 2025-01-09 at 15 28 13" src="https://github.com/user-attachments/assets/5b46d74e-9dc4-4566-9366-d52e22f1f29c" />

but with the trailing slash, it works:

<img width="543" alt="Screenshot 2025-01-09 at 15 29 09" src="https://github.com/user-attachments/assets/ca7a8626-7951-4066-81ae-b6e0c20455c3" />

I'll play with this in a separate PR.